packages/libs/pactjs-test-project/src/example-templates/generated-tx-templates/index.ts

```diff
@@ -2,114 +2,114 @@
 import { buildUnsignedTransaction, ICommandBuilder } from '@kadena/client';
```

Comment:
whoohooo!!! I was always ignoring this file.
or was this by accident?

Reply:
haha this was by accident. I don't know what causes this to get formatted, but it shouldn't be an issue

Reply:
@webpro I think everyone would appreciate someone taking time to look into this haha I thought I changed this as well, but I'll change it back

Reply:
Hopefully this fixes it: https://github.com/kadena-community/kadena.js/pull/342